src/style.css

```diff
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  position: relative;
+}
+
+.search__input {
+  position: relative;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 20px;
+  background-position: 26px;
+  padding-left: 62px;
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-size: 19px;
+  font-size: 16px;
+}
+
+.search-bar::before {
+  content: '';
+  position: absolute;
+  background: url('/src/images/Search.svg') no-repeat;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
```

Comment:
The 'search-bar::before' pseudo-element seems to be creating an icon for the search input. Consider using a more semantic approach with an 'input' with type='search' and a 'label' for accessibility.